### PR TITLE
add function and example to produce file to compute travel times in o…

### PR DESCRIPTION
**Pull request by sannecottaar**
Finally made this little tool to output a layer (with a background 1D seismic model) or planet to a format that can be read by obspy to predict travel times and ray paths. I would like to use this in the cider tutorial (potentially predicting phases for a mars-like planet), so it would be helpful if someone could look at this quickly (the material is due in a week...).

### CODE REVIEW

**Comment by sannecottaar** [comment]
Forgot to mention the example relies heavily on Obspy, so it currently cannot be tested (maybe we want to exclude it from testing?) 

P.s. the rest of the seismic output still needs updating to use Layer or Planet

File: examples/example_seismic_travel_times.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+import matplotlib.pyplot as plt  # Library used for plotting
+# Import BurnMan
+sys.path.insert(1, os.path.abspath('../'))  # add path to burnman
+import burnman
+from burnman import minerals  # import mineral library seperately
+
+
+# This example relies heavily  on the ObsPy, a python seismology toolkit
+import obspy
+from obspy.taup import TauPyModel
```
[the remark this answers is not in the document]
Comment (sannecottaar) [reply examples/example_seismic_travel_times.py]: I've added an exception to the tests instead. I don't like that the try/except statements will avoid proper error messages.

File: examples/example_seismic_travel_times.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+        obspy.taup.taup_create.build_taup_model(
+            "./" + modelname + ".tvel", ".")
+
+        # Arrivals to plot, some random examples of phase names to use ("P","S", "PcP", "Sdiff", "SKS", "PKIKP")
+        # Phase naming in obspy.taup is explained at
+        # https://docs.obspy.org/packages/obspy.taup.html
+        phase_list = ["P", "PKP", "PKIKP"]
+
+        # Time to plot some predictions using routines from Obspy
+        plt.figure(figsize=[9, 7])
```
[the remark this answers is not in the document]
Comment (sannecottaar) [reply examples/example_seismic_travel_times.py]: subplots are a bit more fussy with polar and non-polar ones. But I've made the plotting into a function, and made it a bit more clear what parameters to play around with. Definitely not needed as a general burnman module, as it is meant to be played with in this example (in any case it is also calling obspy modules to do the actual work)

**Comment by sannecottaar** [comment]
Okay, followed most of your suggestions, Bob. Thanks!

File: examples/example_seismic_travel_times.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bobmyhill) [reply examples/example_seismic_travel_times.py]: Fine by me. On the whole, I'd prefer test.sh run the maximum number of tests on any given machine, but there's also something to be said for test consistency.

**Pull request merged**